app/lib/katello/lazy_accessor.rb

```diff
@@ -121,12 +121,7 @@ def lazy_attribute_was(attr)
       end
 
       def lazy_attribute_set(attr, val)
-        attr = attr.to_s
-
-        old = instance_variable_get("@#{attr}").nil? ? self.send(attr) : instance_variable_get("@#{attr}")
-        changed_remote_attributes[attr] = old if old != val
-
-        instance_variable_set("@#{attr}", val)
+        instance_variable_set("@#{attr.to_s}", val)
```

Comment:
Note:  my vcr testing below actually revealed a bug.  When HypervisorsUpdate calls:
```
host.subscription_facet.candlepin_consumer.consumer_attributes = @candlepin_attributes[uuid]
```

this was triggering a get request to candlepin, meaning this is likely much slower than it needs to be.  Discussing with @parthaa, we agreed that the behavior this method is providing was used long long ago when katello used the old queue based orchestration and would use lazy_accessor for updating backed data (all of which was replaced with foreman-tasks).  I don't forsee any harm in removing this, if anything comes up, we probably want to make this change regardless (and fix anything that depends on this).

Reply:
wow, cool! curious what the test results will look like

Reply:
I ended up actually moving this change to another PR: https://github.com/Katello/katello/pull/8024

but this one now depends on it (so save having to rerecord cassettes after merge)

Reply:
Looks like this shaved off about ~20 seconds from a 3 minutes 20 second hypervisor with ~420 hypervisors.